packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -626,6 +597,63 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		}
 	}
 
+	private createRevertible(
```

Comment:
Nit: there is a fair amount of stuff going on in this method. Even though it's private, it still might be worth documenting it.